lua/terrortown/lang/zh_hans.lua

```diff
@@ -2065,49 +2065,52 @@ Propspec 冲刺是向目标矢量方向的移动。它们可以比正常移动
 
 该变量是推力的乘数。]]
 L.label_spec_prop_dash = "冲刺力倍增器"
-L.label_keyhelper_possession_dash = "prop：向视线方向冲刺"
+L.label_keyhelper_possession_dash = "Prop：向视线方向冲刺"
 L.label_keyhelper_weapon_drop = "尽可能丢出所选武器"
 L.label_keyhelper_ammo_drop = "将选定武器的弹药从弹夹中取出"
 
 -- 2023-12-13
---L.vis_no_pickup = "Only public policing roles can pick up the visualizer"
---L.newton_force = "FORCE"
+L.vis_no_pickup = "只有公共警察角色才能捡起显像器"
+L.newton_force = "强制"
```

Comment:
```suggestion
L.newton_force = "推力"
```
Force as in push force, not coercion.